File: Stack.cpp
#include <bits/stdc++.h>

using namespace std;

stack<int> s;

int main(void) {
  s.push(5);
  s.push(2);
  s.push(3);
  s.push(7);
  s.pop();
  while (!s.empty()) {
    cout << s.top() << ' ';
    s.pop();
  }
}

File: DFS_BFS/Stack.py
stack = []

stack.append(5)
stack.append(2)
stack.append(3)
stack.append(7)
stack.pop()
stack.append(1)
stack.pop()

print(stack[::-1]) # 최상단 원소 ~ 최하단 원소
print(stack) # 최하단 원소 ~ 최상단 원소

